### DISCUSSION TITLE
Footnotes?

### DISCUSSION THREAD

**Discussion by einarpersson** [epwalsh/obsidian.nvim#751] [Ideas]
Is there some way to get [footnotes](https://obsidian.rocks/footnotes-in-obsidian/) working with this plugin? E.g. when doing `:ObsidianFollowLink` it should go to footnote.

**Comment by ficd0** [comment]
I think this would be valuable. I'm new to this plugin but I'm excited about contributing so I will look into how the link following works and try to implement this. Would you be interested in helping?
